### PR TITLE
Updated pages across the site

### PR DESCRIPTION
**Pull request by evildmp**
- notes about the general programme
- notes about the Introduction day
- advice on accommodation
- advice on travel
- other minor fixes and changes

### CODE REVIEW

**Comment by evildmp** [comment]
@inglesp ^^^

**Comment by kragniz** [comment]
I had a read over. +1

File: content/accommodation.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+We're negotiating with several local hotels to provide preferential rates for attendees. As we make these arrangements,
+we'll list them here, along with codes you might need to secure a discount. **We hope to have a complete list by Friday
+6th May.**
+
+In the meantime...
+
+### Other hotels
+
+Check sites such as [Hotels.com](http://hotels.com) and [Booking.com](http://booking.com) for other hotels (apparently
+if you put your browser into private or incognito mode, you can sometimes get different, better, prices). Who knows?
```
Comment (inglesp) [inline content/accommodation.md]: Does this "Who knows?" belong inside the parentheses?

File: content/cfp.html [hunk 2]
```diff
@@ -82,6 +82,6 @@
 <p>We’ll publish the list of selected talks as soon as we can after the deadline.</p>
 
 <h2 id="financial-support">Financial support for speakers</h2>
-<p>Grants to assist speakers on low budgets will be available, as well as free and discounted tickets. They'll be allocated by the committee. Any speaker who needs help will receive it. <a href="/financial-aid/">Click here</a> to apply for financial aid. Financial aid applications close when the CFP closes.</p>
+<p>Grants to assist speakers on low budgets will be available, as well as free and discounted tickets. They'll be allocated by the committee. Any speaker who needs help will receive it. <a href="/financial-aid/">Apply for financial assistance.</a> Financial aid applications close when the CFP closes.</p>
```
Comment (inglesp) [inline content/cfp.html:85]: We use "financial aid" elsewhere on the site.  I'm not bothered either way, but if you want to be consistent, you could make changes elsewhere.  @Lukasa may have opinions on this.

File: content/diversity-target.html [hunk 3]
```diff
@@ -1,6 +1,8 @@
 type: page
 title: Diversity Target
 slug: diversity-target
+callout_big_1: We're measurable
```
Comment (inglesp) [inline content/diversity-target.html:4]: I'm not wild about this as a title, but I can't think of anything better beyong "Our target for diversity".

File: content/index.md [hunk 4]
```diff
@@ -1,43 +1,70 @@
 type: page
 slug: index
 url: index.html
-title: Home
+title: PyCon UK 2016
```
Comment (inglesp) [inline content/index.md:4]: You know, I thought I'd done this!

File: content/index.md [hunk 5]
```diff
@@ -1,43 +1,70 @@
...
 callout_big_2: volunteer-run conference
 callout_small: Cardiff City Hall from Thursday 15th to Monday 19th September 2016
 ---
 
 PyCon UK is the annual gathering of the UK Python community and its friends
 from around the world.  This year, the conference is held at [Cardiff City
-Hall](http://www.cardiffcityhall.com/), from Friday 16th to Monday 19th
-September, with a day of introductory workshops on Thursday 15th.
+Hall](http://www.cardiffcityhall.com/), from Thursday 15th to Monday 19th
+September.
```
Comment (inglesp) [inline content/index.md:13]: Good catch!

File: content/index.md [hunk 6]
```diff
@@ -1,43 +1,70 @@
...
+
+## Overview of the programme
```
Comment (inglesp) [inline content/index.md:15]: `h2`s are too big for these headings.  I think `h3`s are a bit too big, but `h4`s are much too small.  Could you change them to `h3`s?

I like breaking things up with headings though.

File: content/index.md [hunk 7]
```diff
@@ -1,43 +1,70 @@
...
+
+* **Thursday**: a day of [introductory workshops, talks and more](/introduction)
+* **Friday to Sunday**: a packed programme of talks and other sessions
+* **Monday**: sprints and more advanced workshops
```
Comment (inglesp) [inline content/index.md:19]: Could we include hackathons in this list?

File: content/introduction_day.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+## It's for...
+
+### Beginners
+
+
+We'll have workshops on a variety of topics, and talks suitable for those new to programming. The day will help give
+newcomers a head start, and make the following days' talks and activities more useful and accessible to them.
+
+If you ever felt that a PyCon might be fun but that you weren't expert enough to attend - this day is guaranteed
```
Comment (inglesp) [inline content/introduction_day.md]: s/If you/If you've/

File: content/introduction_day.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+### Talks
+
+We'll have talks on a variety of subjects - [perhaps yours too?](/cfp)
+
+### And also...
+
+You might like to introduce a new idea, plan or initiative to the conference. It's our experience that giving people a
+chance to think and talk about it for a few days before coming together in earnest for some more concerted efforts
+later on pays dividends. A meeting or gathering on Thursday would be an excellent time to do it. All you need to do is
+[tell us about your suggestion](/cfp).
```
Comment (inglesp) [inline content/introduction_day.md]: I don't really understand this section.

File: content/introduction_day.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+
+## Help out!
+
+Your experience, insight, patience, empathy and hard-won successes will be of enormous value at the Introduction day.
+Please share them - as a helper in a workshop, by giving a talk, by running a clinic, or something else entirely. We
+don't always get the chance to enjoy new people take some of the most rewarding and exciting steps in programming - so
```
Comment (inglesp) [inline content/introduction_day.md]: s/take/taking/

File: content/venue.md [hunk 11]
```diff
@@ -9,10 +9,56 @@ The main conference venue is Cardiff’s beautiful [City
 Hall](http://www.cardiffcityhall.com/), with evening events held around the
 centre of Cardiff.
 
-### Getting to the venue
+See [Accommodation](/accommodation) for recommended hotels and other places to stay.
```
Comment (inglesp) [inline content/venue.md:12]: s/Accommodation/the accommodation page/

File: content/venue.md [hunk 12]
```diff
@@ -9,10 +9,56 @@ The main conference venue is Cardiff’s beautiful [City
...
+CF10 3ND. Cardiff City Halls is accessible for people of limited mobility. If you need to park near the venue because
+of mobility issues, please contact us and we'll do our best to help.
+
+## Getting to Cardiff
+
+### Trains
+
+Trains to Cardiff arrive right in the city centre, at Cardiff Central railway station. Unfortunately, engineering work
+at the Severn Tunnel means that trains around the time of the conference will be disrupted, and some parts of affected
+journeys may be replaced by bus services.
```
Comment (inglesp) [inline content/venue.md:26]: Bugger.

File: content/venue.md [hunk 13]
```diff
@@ -9,10 +9,56 @@ The main conference venue is Cardiff’s beautiful [City
...
+Birmingham Airport is about two hours away by train. The trip requires a change at Birmingham New Street station.
+
+Flights from London Heathrow have reasonable connections by bus. Other London airports are further away. Avoid flying
+to London Stansted or Luton airports unless you have to. Getting from there to Cardiff will be an unrewarding
+experience.
+
+### Cars
+
+There's limited parking in the city centre, and you'll pay for the privilege. However, if you need to use a car because
+of mobility issues, please contact us.
```
Comment (inglesp) [inline content/venue.md:64]: This is all really good.

File: content/accommodation.md (hunk 1 above)
Comment (evildmp) [inline content/accommodation.md]: Willfix

File: content/cfp.html (hunk 2 above)
Comment (evildmp) [inline content/cfp.html:85]: Much prefer _assistance_ over _aid_, which has connotations of vulnerability or even desperation that might make some people feel more reluctant to ask for it (first aid, medical aid, foreign aid, Live Aid, etc). There's not much in it, but I do think that someone who asks for assistance is in a slightly different position from someone who asks for aid.

File: content/diversity-target.html (hunk 3 above)
Comment (evildmp) [inline content/diversity-target.html:4]: Our targets are measurable. We're giving people something to judge us by.

File: content/index.md (hunk 6 above)
Comment (evildmp) [inline content/index.md:15]: If they are too big, the CSS needs to change, not the level of headings. `h1` is reserved in the header, and the next level should be `h2`. Semantics!

File: content/introduction_day.md (hunk 9 above)
Comment (evildmp) [inline content/introduction_day.md]: Improved.

File: content/venue.md (hunk 12 above)
Comment (evildmp) [inline content/venue.md:26]: Indeed. They chose our precise weekend - and announced it after we announced ours!

**Pull request merged**